wagtail/research/comparisons-to-existing-tools.md

```diff
@@ -0,0 +1,19 @@
+# Comparisons to existing tools
+
+Over the past year DataMade has employed two CMS platforms:
+
+- [Django CMS](https://www.django-cms.org/en/) for [Neighborhood Opportunity Fund](https://github.com/datamade/neighborhood-fund)
+- [Wagtail](https://wagtail.io/) for [LISC CDNA](https://github.com/datamade/lisc-cnda) and the [Lugar Center Oversight Index](https://github.com/datamade/committee-oversight)
+
+
+## Wagtail vs. Django CMS
```

Comment:
It might also be nice to compare Wagtail to Django's default admin interface. What advantages does it offer?

Reply:
I agree, but I've never used Django's default admin interface in a project so I don't feel prepared to answer that question. Is that something you'd be interested in adding a section for here?

Reply:
Sure!